ansible_collections/arista/avd/molecule/eos_designs_unit_tests/inventory/group_vars/AUTOVPN_TESTS.yml

```diff
@@ -100,8 +100,7 @@ tenants:
 
 wan_virtual_topologies:
   vrfs:
-    - name: default
-      policy: DEFAULT-AVT-POLICY
+    # Not configuring default VRF policy on purpose to test auto-generation
```

Comment:
Why are these changes causing the IT stuff to get removed?

Reply:
this is covered in CV PATHFINDER

Reply:
otherwise I need two new examples in eos_designs just for this

Reply:
You could just add another VRF with the old policy, so we keep all the checks also for autovpn case.

Reply:
adding new hosts instead for thsi specifically